dashboard/app/helpers/aichat_openai_helper.rb

```diff
@@ -69,6 +69,30 @@ def self.request_chat_completion(messages, temperature)
     [response, body&.dig('usage')]
   end
 
+  def self.request_structured_chat_completion(messages, tools:, tool_choice:, temperature:)
+    http_response = client.request_chat_completion(
```

Comment:
Is there an update to `OpenaiChatHelper` that's missing in this diff? Presumably we'd also have to update the `request_chat_completion` function to accept the `extra` arg right?